app.js

```diff
@@ -6,10 +6,10 @@ var Server;
 var http = require('http');
 var https = require('https');
 
-//Read JSON object
-console.log(config);
+// Export config object
+exports.module={config};
```

Comment:
As informed in below Review Comments. Do no require App.js in storage/index rather require storage module here and pass config in init as argument.